src/Altinn.Apps/AppTemplates/AspNet/Altinn.App.Api/Controllers/ApplicationMetadataController.cs

```diff
@@ -52,5 +54,82 @@ public IActionResult GetAction(string org, string app)
 
             return NotFound();
         }
+
+        /// <summary>
+        /// Get the application XACML policy file
+        ///
+        /// If org and app does not match, this returns a 409 Conflict response
+        /// </summary>
+        /// <param name="org">Unique identifier of the organisation responsible for the app.</param>
+        /// <param name="app">Application identifier which is unique within an organisation.</param>
+        /// <returns>XACML policy file</returns>
+        [HttpGet("{org}/{app}/api/v1/policy.xml")]
+        public IActionResult GetPolicy(string org, string app)
+        {
+            Application application = _appResources.GetApplication();
+            string policy = _appResources.GetApplicationXACMLPolicy();
+
+            if (application != null && policy != null)
+            {
+                string wantedAppId = $"{org}/{app}";
+
+                if (application.Id.Equals(wantedAppId))
```

Comment:
I think we can just remove this validation. In real world we would not hit the incorrect endpoint.
And then you can remove the getappid endpoint. (you anyway does not have the correct appid for that either)

Reply:
Yeah. Ideally we should get rid of these in applicationmetadata.json since they are "duplicate" info where this file is not the real master.

```json
  "id": "org/app",
  "org": "org",
  "created": "2019-09-16T22:22:22",
  "createdBy": "username",
  "title": { "nb": "Bestillingseksempelapp" },
```

Reply:
So, how would you get appId in Startup.Configure for swagger and Staticfiles?

Reply:
I think anyway that is out of scope for this change, and I am also not sure we can remove this completly